submissions/random_forest_100/estimator.py

```diff
@@ -0,0 +1,7 @@
+from sklearn.ensemble import RandomForestRegressor
+
+
+def get_estimator():
+    pipeline = RandomForestRegressor(
+        n_estimators=50, max_leaf_nodes=3, random_state=61)
+    return pipeline
```

Comment:
don't call this pipeline plz

maybe just "reg = RandomForestRegressor(...)"

I find it misleading that you call it pipeline.